### DISCUSSION TITLE
Rumble, Odysee and Dlive Compatibility

### DISCUSSION THREAD

**Discussion by AlphaPyxis** [3dproger/AxelChat#200] [Ideas]
Hey,

Just came across this chat app and would be perfect only it doesnt have compatibility for rumble, Odysee or Dlive. Id really love compatibility for these especially Rumble as i am trying to make that my main platform but for now i am streaming to twitch and Rumble and its annoying using two chats. Also is there a relayer built into this chat app? like will messages be relayed between platfroms so each stream can see what each viewer is saying? Thank you in advance for the help and reply!

**Comment by 3dproger** [comment]
Hello,

As development progresses, I will gradually add support for the Rumble, Odysee and Dlive platforms as soon as the opportunity and time arise. Rumble support will be a higher priority. The relayer, if it will be implemented, will not be soon, since this is a more labor-intensive task. Thanks for the ideas!

**Reply by AlphaPyxis** [reply]
Hello,

Thank you for the response. Awesome glad to hear that rumble will be higher priority as i am really enjoying that platform over the others and am definitely getting better interaction on it. Awe that's a pity about the relayer. Is there not existing codebase for such a thing out there that can be modified to work with your program? Sorry for the question I am not a programmer and don't know the process. Thank you for the response!

**Reply by 3dproger** [reply]
Hello,

The relayer codebase exists, but its integration into AxelChat can take a very long time, because I will have to make a own relayer for each streaming platform, so for now this idea is postponed until indefinitely

**Comment by 3dproger** [comment]
Hello! I made Rumble support in the latest update 0.26.0

**Reply by AlphaPyxis** [reply]
Hey man! Awesome work, Tested it earlier it is working great! Although when a mod removes a message from chat it stays on your app. That is the only issue I have observed!

**Reply by 3dproger** [reply]
Glad you liked the result! And with the deletion of messages, I will try to deal with this issue, but so far it is difficult to do

**Reply by AlphaPyxis** [reply]
No problem man I am sure it is a big task! Don't worry too much about it i am just providing feedback as I see things crop up. Oh one question. In the "viewer counter" widget what is the Font used for the digit. I have to use a separate browser source for DLive and want to match the font :) Thank you in advance I am really enjoying your app. I love the notification sound when I get a message. It ensures I don't miss a message! Keep up the great work!

**Reply by 3dproger** [reply]
Depending on the system and browser, `Segoe UI` or `Roboto` font will most likely be used. But in the future, I plan to add the ability to customize the appearance, including changing the font

**Reply by AlphaPyxis** [reply]
Nice thank you for the answers, I am sorry about all the questions but I have been able to use the widget and using some CSS was able to modify the CSS file to change text colour in the Chat widget. Would I be able to somehow change it so it shows one message at a time within a specified character limit and hide the platform symbol and avatar so its just the name and the message and one message at a time or is this too complex? I don't even know where to begin with it as it took me a little to figure out where the text colour was in the file. Again thanks for this great software and for your continued hard work!

**Comment by 3dproger** [comment]
At the moment, it is very difficult to do all this through CSS. But soon I will release an update, where there will be customization of the appearance of widgets without having to dig into css. While it will be possible to change the colors and sizes of texts, as well as turn off the display of avatars and platform icons. In the future, I will expand the possibilities of customization

**Reply by AlphaPyxis** [reply]
Awesome man thank you for the great answer to my questions I eagerly await this update! Have a great rest of your weekend!

**Comment by 3dproger** [comment]
Hello! In update 0.31.0, I made integration with Odysee. Haven't made the display of emotes, stickers and some badges yet. And maybe there are some other little problems. All current integration issues will be displayed here https://github.com/3dproger/AxelChat/issues?q=is%3Aissue+is%3Aopen+label%3AOdysee

**Comment by 3dproger** [comment]
Hello! The new version now has compatibility with DLive

**Reply by AlphaPyxis** [reply]
Awesome man! Thanks so much for these great updates, You are a legend. Quick question. Would there be a way for you to implement a way to be able to output the "sound commands" to particular devices on your computer. I have Virtual cables and would prefer to use a specific output device for the sounds on axelchat. Keep up the great work man :)

**Reply by 3dproger** [reply]
Good idea, I'll try to make it possible to customize the audio output device in the future. Thank you :)

**Comment by 3dproger** [comment]
Well, I'll close this thread then, as it's not relevant. If you have any more questions, you can open a new one
